Tests/XCUITests/PhotonActionSheetTest.swift

```diff
@@ -34,35 +34,36 @@ class PhotonActionSheetTest: BaseTestCase {
     func testShareOptionIsShown() {
         navigator.openURL(path(forTestPage: "test-mozilla-org.html"))
         waitUntilPageLoad()
-        waitForExistence(app.buttons[AccessibilityIdentifiers.Toolbar.shareButton], timeout: 10)
-        app.buttons[AccessibilityIdentifiers.Toolbar.shareButton].tap()
+        navigator.goto(BrowserTabMenu)
+        waitForExistence(app.tables["Context Menu"].otherElements[ImageIdentifiers.share], timeout: TIMEOUT)
+        navigator.performAction(Action.ShareBrowserTabMenuOption)
 
         // Wait to see the Share options sheet
-        waitForExistence(app.buttons["Copy"], timeout: 10)
+        waitForExistence(app.buttons["Copy"], timeout: TIMEOUT)
     }
 
     // Smoketest
     func testShareOptionIsShownFromShortCut() {
         navigator.openURL(path(forTestPage: "test-mozilla-org.html"))
         waitUntilPageLoad()
-        waitForExistence(app.buttons[AccessibilityIdentifiers.Toolbar.shareButton], timeout: 10)
```

Comment:
XCode keeps complaining about the `shareButton`.  It looks like this is introduced via a change after this branch was created.